R/oc_forward.R

```diff
@@ -256,6 +262,10 @@ oc_forward <-
 #' @param abbrv Logical vector, or an unquoted variable name of such a vector.
 #'   Default is `FALSE`. When `TRUE` addresses in the `oc_formatted` variable of
 #'   the results are abbreviated (e.g. "Main St." instead of "Main Street").
+#' @param address_only Logical vector, or an unquoted variable name of such a
```

Comment:
why is the param not inherited?

Reply:
Because the docs are slightly different (e.g. "or an unquoted variable name of such a vector") because the _df functions accept data.frames as input. Not saying that this is a particularly good reason...

Reply:
makes sense! I was just curious :-)